pkg/certificate/authority.go

```diff
@@ -6,14 +6,10 @@ import (
 	"fmt"
 	"path/filepath"
 
+	"github.com/massalabs/station/int/configuration"
```

Comment:
here in pkg/certificate we are importing int/configuration so when package certificate depends on int of massa station. So we can't extract package certificate.

I had in mind we pass as an argument to CA::Load the CAPath.

Reply:
okay